docs/_themes/clusterhq/layout.html

```diff
@@ -0,0 +1,192 @@
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+    <script type="text/javascript">
+        var DOCUMENTATION_OPTIONS = {
+            URL_ROOT:'{{ url_root }}',
+            VERSION:'{{ release|e }}',
+            COLLAPSE_INDEX:false,
+            FILE_SUFFIX:'{{ '' if no_search_suffix else file_suffix }}',
+            HAS_SOURCE:  {{ has_source|lower }}
+        };
+    </script>
+    {%- for scriptfile in script_files %}
+        <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
+    {%- endfor %}
+    {%- if pagename == "search" %}
+    <script type="text/javascript">
+        jQuery(function() { Search.loadIndex("searchindex.js"); });
+    </script>
+    {%- endif %}
+    <link rel="top" title="{{ docstitle|e }}" href="{{ pathto('index') }}"/>
+    
+    {% if theme_analytics_code %}
+        <script type="text/javascript">
+            var _gaq = _gaq || [];
+            _gaq.push(['_setAccount', '{{ theme_analytics_code }}']);
+            {% if theme_analytics_domain %}
+                _gaq.push(['_setDomainName', '{{ theme_analytics_domain }}']);
+            {% endif %}
+            _gaq.push(['_trackPageview']);
+
+            (function () {
+                var ga = document.createElement('script');
+                ga.type = 'text/javascript';
+                ga.async = true;
+                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+                var s = document.getElementsByTagName('script')[0];
+                s.parentNode.insertBefore(ga, s);
+            })();
+        </script>
+    {% endif %}
+</head>
+<body>
+    
+    <!-- Menu -->
+    <div class="menubar">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-12 col-md-3 col-xs-6 logo">
+                    <a href="https://clusterhq.com/">
+                        <img src="{{ pathto('_static/logo@2x.png', 1) }}" width="200" height="50">
+                    </a>
+                </div>
+                <div class="visible-xs-block col-xs-6 mob-nav">
+                    <a href="javascript:$('nav ul').toggleClass('hidden-xs').toggleClass('visible-xs');">
+                        <img src="{{ pathto('_static/menu@2x.png', 1) }}" width="60" height="47" class="pull-right"/>
+                    </a>
+                </div>
+                <nav class="clearfix">
+                    <div class="col-sm-12 col-md-9">
+                        <ul class="menu hidden-xs">
+                            <li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/index.html">Getting Started</a></li>
```

Comment:
So do you want to block merging this on that? Or add a note to that issue to change this link.